packages/client/src/router.tsx

```diff
@@ -0,0 +1,42 @@
+import { createBrowserRouter } from 'react-router-dom'
+
+const router = createBrowserRouter([
+  {
+    path: '/',
+    element: <p>Cтраница входа и регистрации</p>,
+  },
+  {
+    path: '/main',
+    element: <p>Главная страница</p>,
+  },
+  {
+    path: '/game',
+    element: <p>Страница игры</p>,
+  },
+  {
+    path: '/profile',
+    element: <p>Cтраница профиля</p>,
+  },
+  {
+    path: '/rating',
+    element: <p>Cтраница лидерборда</p>,
+  },
+  {
+    path: '/forum',
+    element: <p>Страница форума</p>,
+  },
+  {
+    path: '/forum-top',
```

Comment:
Страница топика - это скорее всего внутренная страница форума, т.е. тут динамический  path должен быть, чтоб можно было отображать разные топики
```suggestion
    path: '/forum/:topicId',
    Component: () => {
      const {topicId} = useParams();
      return <p>Страница форума, тема {topicId}</p>
    },
```